tests/docker-compose.yml

```diff
@@ -60,4 +60,3 @@ services:
         TERM: xterm
     depends_on:
       - conjur
-      - conjur_cli
```

Comment:
Woops I misread the diff in my previous comment. You can leave the conjur_cli dependency and just address the missing newlines. There's still one in `tests/test.sh`